docs/content/preview/benchmark/sysbench-ysql.md

```diff
@@ -47,47 +71,30 @@ Start your YugabyteDB cluster by following the steps in [Manual deployment](../.
 You will need the IP addresses of the nodes in the cluster for the next step.
 {{< /tip>}}
 
-### Run the benchmark
-
-Run the `run_sysbench.sh` shell script to load the data and run the various workloads:
-
-```sh
-./run_sysbench.sh --ip <ip>
-```
-
-This script runs all 8 workloads using 64 threads with the number of tables as 10 and the table size as 100k.
-
-If you want to run the benchmark with a different count of tables and tablesize, do the following:
-
-```sh
-./run_sysbench.sh --ip <ip> --numtables <number of tables> --tablesize <number of rows in each table>
-```
-
-### Run individual workloads (optional)
+### Run individual workloads
 
 You can choose to run the following workloads individually:
 
-* oltp_insert
-* oltp_point_select
-* oltp_write_only
 * oltp_read_only
 * oltp_read_write
+* oltp_multi_insert
 * oltp_update_index
 * oltp_update_non_index
 * oltp_delete
 
+
 Before starting the workload, load the data as follows:
 
 ```sh
-$ sysbench <workload>               \
-      --tables=10                   \
-      --table-size=100000           \
-      --range_key_partitioning=true \
-      --db-driver=pgsql             \
-      --pgsql-host=127.0.0.1        \
-      --pgsql-port=5433             \
-      --pgsql-user=yugabyte         \
-      --pgsql-db=yugabyte           \
+$ sysbench <workload>               			\
```

Comment:
alignment of slashes is off

Reply:
Should I correct it? I'm asking because that's how it was written in the Google doc.

Reply:
will correct the alignment